### DISCUSSION TITLE
Prisma 2 Fixtures Loader

### DISCUSSION THREAD

**Discussion by nmacherey** [prisma/prisma#4865] [Q&A]
Hi,

First of all thanks for your work ! 

I've been looking for a fixtures loading tool for Prisma2 and I found nothing. So I decided to adapt a fixtures loader tool have been using with typeorm for Prisma2. 

It can be found here https://github.com/getbigger-io/prisma-fixtures I kept the original MIT license. 

Regards

**Comment by ryands17** [comment]
Hey @nmacherey 👋 
Thanks a lot for creating this package! Looks great and will try this out as well :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
